packages/palette_generator/lib/palette_generator.dart

```diff
@@ -150,6 +172,28 @@ class PaletteGenerator with Diagnosticable {
       'The encoding must be RGBA with 8 bits per channel.',
     );
 
+    // If we want to use an isolate, make a recursive compute call. We do it
+    // this way to avoid changing the rest of the function.
+    if (useIsolate) {
+      return compute(
+          (_FromByteDataArgs message) => fromByteData(
+                message.encodedImage,
+                region: message.region,
+                maximumColorCount: message.maximumColorCount,
+                filters: message.filters,
+                targets: message.targets,
+                // This is false to stop us from infinitely recursing
```

Comment:
Missing a period. https://dart.dev/effective-dart/documentation#do-format-comments-like-sentences

Reply:
Fixed in 4992c3b98.